spec/system/consumer/white_label_spec.rb

```diff
@@ -228,6 +228,11 @@
           it "does not show the OFN logo on shop page" do
             expect(page).not_to have_selector "img[src*='/default_images/ofn-logo.png']"
           end
+          it "link the logo to the default url" do
```

Comment:
:gb: :teacher: :wink: 
```suggestion
          it "links the logo to the default URL" do
```